src/display.rs

```diff
@@ -236,7 +236,6 @@ impl Display {
     /// Currently this only works on X11 with XComposite and XDamage extensions available.
     ///
     /// Deprecated since GDK 3.16
-    #[cfg(not(feature = "gdk_3_16"))]
```

Comment:
When the features were set manually, this attribute made some sense, though I'd still disagree with removing functions that have been deprecated so recently. With automatic detection it would set the users up for surprising breakage caused by simply updating their libraries -- a big no-no.